bigtable-hbase-2.x-parent/bigtable-hbase-2.x-integration-tests/src/test/java/com/google/cloud/bigtable/hbase/async/TestAsyncAdmin.java

```diff
@@ -152,12 +153,11 @@ public void testGetTableDescriptor_nonExistingTable() throws Exception {
 
   @Test
   public void testGetTableDescriptor_nullTable() throws Exception {
-    // This breaks the minicluster, for some reason.
-    if (SharedTestEnvRule.getInstance().isBigtable()) {
-      AsyncAdmin asyncAdmin = getAsyncConnection().getAdmin();
-      assertEquals(null, asyncAdmin.getDescriptor(null).get());
-    }
-  }  
+    // This breaks the minicluster, for some reason, so only run it for Cloud Bigtable.
+    Assume.assumeTrue(SharedTestEnvRule.getInstance().isBigtable());
```

Comment:
Please add a message for the the assumption

Reply:
done